### DISCUSSION TITLE
Explicit allow option for Wi-Fi connections

### DISCUSSION THREAD

**Discussion by bcelary** [borgbase/vorta#1654] [Ideas]
My Vorta runs daily backups when I am at home to my local NAS server. But I often move around and any other WiFi should not be used to even attempt a backup.

Would you consider adding an option that would require user to explicitly add a WiFi to allowed networks? Especially when using local address (like 192.168... or host.local name)? Also the message that WiFi is not allowed might become a non-error message, as Vorta keeps nagging me all the time when I am away from home.

These options could be added in the `Networks` panel, here:
![image](https://user-images.githubusercontent.com/376943/225550787-63a7e8ce-477a-4cc4-a605-94b83d9e4cbf.png)

E.g. `x Don't allow new WiFi connections` and `x silence "cuurent WiFi is not allowed" messages`.

Asking if such idea makes sense as this seems like something easy to add and I might as well create a PR for this.

**Comment by real-yfprojects** [comment]
I opened a feature request: #1620

**Reply by bcelary** [reply]
Awesome!

Was the number #1658 ?
